content/nomad/v1.11.x/content/docs/configuration/client.mdx

```diff
@@ -520,6 +525,39 @@ refer to the [drivers documentation](/nomad/docs/job-declare/task-driver).
   the Nomad client's environment. By default a minimal environment is set including
   a `PATH` appropriate for the operating system.
 
+### `fingerprint` Parameters
+
+- `name` `(string: "")` - The name of the fingerprinter the configuration
+  applies to. This field is required and is provided as part of the block label.
+  It supports `"env_aws"`, `"env_azure"`, `"env_digitalocean"`, and `"env_gce"`.
+
+- `retry_attempts` `(int: 0)` - Specifies the maximum number of retry attempts
+  to make before giving up. This field supports `-1` for unlimited retries, but
+  use is not recommended as this may cause the client to hang indefinitely on
```

Comment:
```suggestion
  to make before giving up. This field supports `-1` for unlimited retries, but
  we do not recommend `-1` as this may cause the client to hang indefinitely on
```
style nit: use "we" when stating what HashiCorp recommends